app/controllers/forest_liana/resources_controller.rb

```diff
@@ -157,14 +159,24 @@ def render_jsonapi getter
       records = getter.records.map { |record| get_record(record) }
       fields_to_serialize = fields_per_model(params[:fields], @resource)
 
+      meta = {
+        count: getter.count
+      }
+
+      if params[:search]
+        meta[:decorators] = decorateForSearch(records, getter.search_query_builder.fields_searched, params[:search])
+      end
+
       json = serialize_models(
         records,
         include: includes(getter),
         fields: fields_to_serialize,
-        count: getter.count,
+        meta: meta,
         params: params
       )
 
+
+
```

Comment:
If you had reviewed your own code, you would have removed this.

Reply:
Fixed 😱